features/step_definitions/check_your_answers_steps.rb

```diff
@@ -100,7 +100,7 @@ def expect_matching_questions_and_answers(actual_selector:, expected_table:)
   expected = expected_table.hashes.map(&:symbolize_keys)
   actual = actual_questions_and_answers_in(selector: actual_selector)
 
-  expect(actual).to match_array(expected)
+  expect(actual).to match_array(expected), SuperDiff.diff(expected, actual)
```

Comment:
SuperDiff.diff? Is this a new pattern?

Reply:
We already use super diff which integrates into rspec and gives a diff when an error is raised. However in this instance in cucumber output it was not giving a diff and i was struggling to see the failure. Adding this output the actual diff (which immediately showed me I had neglected to add a `)` to a translation which i was then able to fix 🥳 . 

in short, it is helps when using this cucumber wrapper step expectation so i left it in